packages/openactive-integration-tests/CONTRIBUTING.md

```diff
@@ -0,0 +1,273 @@
+* `testOpportunityCriteria`: The **Opportunity Criteria** to use for the primary OrderItem under test. For info on criteria, see [Opportunity Criteria](#opportunity-criteria).
+* `controlOpportunityCriteria`: The **Opportunity Criteria** to use for any "control" OrderItems.
+
+    When the test is run with multiple opportunities, a "control" opportunity will be added. With this, one can test that the feature still works if combined with other opportunities.
+
+    For example, if testing an error, set the controlOpportunityCriteria to TestOpportunityBookable to ensure that the correct error is returned even though one of the opportunities in the Order is valid.
+
+    If a control isn't appropriate for a particular test, this can be set to the same value as `testOpportunityCriteria`.
+
+This is not an exhaustive list of properties which `FeatureHelper.describeFeature(..)` accepts. See the code documentation for the full picture.
+
+For more information on `FeatureHelper`, see [the ARCHITECTURE doc](./ARCHITECTURE.md#feature-helper).
+
+### Opportunity Criteria
+
+What a Criteria is is documented in [the test-interface-criteria docs](../test-interface-criteria/README.md#criteria).
+
+Each test is configured to only run for Opportunities which conform to some criteria. This is primarily specified by the `testOpportunityCriteria` property of `FeatureHelper.describeFeature(..)`.
+
+In this example, the test will only run for Opportunities which are bookable, cancellable, and where the current time falls within the cancellation window:
+
+```js
+FeatureHelper.describeFeature(module, {
+  testName: 'Successful booking and cancellation within window.',
+  testDescription: 'A successful end to end booking including cancellation within the cancellation window.',
+  testOpportunityCriteria: 'TestOpportunityBookableCancellableWithinWindow',
+```
+
+This is particularly useful for this test, as it checks that these types of Opportunities can be successfully booked and cancelled. If the test were to run for Opportunities which are not cancellable, then the test would fail, but this would not be a valid failure.
+
+## Writing a new test
+
+### 1. New feature?
+
+If creating a new feature, create a new folder within `test/features/` with the name of the feature. Within that folder, create a `feature.json` file as described in [feature.json](#featurejson).
+
+If this new test is being added to an existing feature, then skip this step.
+
+### 2. New Test Interface Criteria?
+
+Each test has a set of Criteria configured (see [Opportunity Criteria](#opportunity-criteria)). Check whether or not this test can use a Criteria that already exists or if a new one is needed. If a new Criteria is needed, be sure to create it in the [Test Interface](https://openactive.io/test-interface/) spec, as well as in the [`test-interface-criteria`](../test-interface-criteria/) package.
```

Comment:
I haven't gone into more detail here about how to create a new one. For the Test Suite part, there is an issue for that here: https://github.com/openactive/openactive-test-suite/issues/554. Not sure if we need documentation for the Test Interface side